server/graphql/v2/object/Collective.js

```diff
@@ -10,6 +10,34 @@ export const Collective = new GraphQLObjectType({
   fields: () => {
     return {
       ...AccountFields,
+      balance: {
+        description: 'Amount of money in cents in the currency of the collective currently available to spend',
+        type: GraphQLInt,
+        resolve(collective, _, req) {
+          return req.loaders.collective.balance.load(collective.id);
+        },
+      },
+      host: {
+        description: 'Get the host collective that is receiving the money on behalf of this collective',
+        type: Account,
+        resolve: resolveHost,
+      },
     };
   },
 });
+
+function resolveHost(collective, _, req) {
```

Comment:
I see that this logic is copy/pasted from somewhere else.

I would extract that in some shared utility between v1/v2, similar to loaders https://github.com/opencollective/opencollective-api/blob/master/server/graphql/loaders.js which are also share between v1 and v2.

Reply:
Done. There is a `common` folder shared by v1 and v2.